osu.Game/Storyboards/Drawables/DrawableStoryboardAnimation.cs

```diff
@@ -118,8 +118,24 @@ private void load(TextureStore textureStore, Storyboard storyboard)
             for (int frame = 0; frame < Animation.FrameCount; frame++)
             {
                 string framePath = Animation.Path.Replace(".", frame + ".");
+                Drawable sprite = storyboard.CreateSpriteFromResourcePath(framePath, textureStore);
 
-                AddFrame(storyboard.CreateSpriteFromResourcePath(framePath, textureStore), Animation.FrameDelay);
+                if (sprite != null)
```

Comment:
The "extra frames without matching sprites" crash is resolved by the null checks